SystemServices/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [2.1.1] - 2024-03-28
```

Comment:
Need to update the macro also - https://github.com/rdkcentral/rdkservices/blob/49862bdf0ec695851848404039cfad80f6c979d1/SystemServices/SystemServices.cpp#L70

Reply:
Updated as requested.